src/modality-specific-files/physiological-recordings.md

```diff
@@ -72,122 +194,604 @@ A guide for using macros can be found at
+column called `"timestamp"`:
+
+```JSON
+{
+    "SamplingFrequency": 100.0,
+    "StartTime": -22.345,
+    "Columns": ["timestamp", "cardiac"],
+    "cardiac": {
+        "Description": "continuous pulse measurement",
+        "Units": "mV"
+    },
+    "timestamp": {
+        "Description": "a continuously increasing identifier of the sampling time registered by the device",
+        "Units": "ms",
+        "Origin": "System startup"
+    }
+}
+```
+
+The decompressed contents of `sub-01_task-nback_physio.tsv.gz`
+are now:
+
+```{.text linenums="1"}
+10.1    13894432329
+10.0    13894432330
+9.5     13894432331
+9.2     13894432332
+9.0     13894432333
+10.2    13894432334
+10.3    13894432335
+10.1    13894432336
+```
+
+Then, the `sub-01_task-nback_physioevents.json` MAY define
+the `ForeignIndexColumn` for indexing:
+
+```JSON
+{
+    "Columns": ["onset", "message"],
+    "Description": "Messages logged by the measurement device",
+    "ForeignIndexColumn": "timestamp"
+}
+```
+
+The decompressed contents of the corresponding TSV file,
+`sub-01_task-nback_physioevents.tsv.gz` could read now:
+
+```TSV
+13894432325    "Ready"
+13894432331    "Synchronous recalibration triggered"
+13894432334    "External message received: new block"
+```
+
+## Specific physiological signal types
+
+### Eye-tracking
+
+<!--
+1. Datasets will be updated later to adapt to the agreed format.
+2. We aim at adding a last example converting published dataset from openeuro.
+-->
+!!! example "Example datasets"
+
+    [Example datasets](https://bids-standard.github.io/bids-examples/#dataset-index)
+    with eye-tracking data have been formatted using this specification
+    and can be used for practical guidance when curating a new dataset:
+
+    -   Combined behavior and eye-tracking fixation and saccade data,
+        measured with an Eyelink (SR Research), from 8 participants reading 320
+        embedded target words and invisible boundary (see
+        [Gagl, 2016](https://peerj.com/articles/2467/)).
+
+        [BIDS dataset](https://zenodo.org/records/1228659)
+
+    -   Combined behavior and eye-tracking position and pupil data, measured with
+        an Eyelink (SR Research), from 26 participants performing a
+        binocular rivalry task (see
+        [Brascamp et.al, 2021](https://doi.org/10.7554/eLife.66161)).
+
+        [BIDS dataset](https://doi.org/10.5061/dryad.41ns1rncp)
+
+    -   Combined resting-state fMRI and eye-tracking data, measured with an Eyelink
+        (SR research), from 20 participants keeping their gaze steady at the
+        screen center.
+
+        [BIDS dataset](https://openneuro.org/datasets/ds004158/versions/2.0.1)
+
+Setting `PhysioType` to the keyword `"eyetrack"` specifies that
+the physiological recordings in the `<matches>_physio.tsv.gz` have
+been acquired with an *eye-tracker*.
+In the following, *eye-tracker* refers to the apparatus allowing
+the recording of gaze position and/or pupil size.
```

Comment:
```suggestion
the recording of gaze position, and, optionally, pupil size.
```

The `physio.PhysioEyeTracking` columns table that appears below specifies that gaze data are REQUIRED but pupil data are OPTIONAL.  So If I understand the sentence above, "_gaze position and/or pupil size"_ is incorrect because pupil size is not allowed on its own.

Alternatively, we could change the spec to allow pupil size data without eyegaze data (no strong feelings here!)